opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp

```diff
@@ -474,6 +474,9 @@ class Well {
     double getGuideRateScalingFactor() const;
 
     bool hasBeenDefined(size_t timeStep) const;
+    bool hasWlist(const std::string& name) const;
```

Comment:
I will look more closely on this later today - but my initial reaction is that it seems weird that the wells get WLIST functionality?

Reply:
> my initial reaction is that it seems weird that the wells get WLIST functionality?

It does look weird, but there's a reason for it.  In the restart file, the wells know which well lists they're a part of rather than the other way around.

Reply:
> the wells know which well lists they're a part of rather than the other way around.

I understand - I would guess the `WListManager` could supply that information with a well name lookup?

Reply:
OK - I have looked more into this and I am quite certain you can the per well information out from the ẀListManager`with something like:
```C++
std::vector<std::pair<std::string, std::size_t>> WListManager::restart_info(const std::string& wname) {
     ....
}
```

Reply:
This has been changed as requested - moved to the WListManager.